fedimint-server/src/consensus/server.rs

```diff
@@ -508,18 +514,15 @@ impl ConsensusServer {
         self.modules.decoder_registry()
     }
 
-    pub async fn build_session_outcome(&self) -> SessionOutcome {
-        let items = self
-            .db
+    pub async fn pending_accepted_items(&self) -> Vec<AcceptedItem> {
```

Comment:
Technically this breaks SemVer, but nobody is supposed to directly rely on this (we should make it private on master).